src/entities/activity/ui/ConsentPublicModel.tsx

```diff
@@ -0,0 +1,29 @@
+import { useState } from "react"
+
+import { useCustomTranslation } from "~/shared/utils"
+import CustomModal from "~/widgets/Modal"
+
+export const ConsentPublicModel = () => {
```

Comment:
Please consider the name: ConsentPublicMod**a**l

Reply:
Done